### PR TITLE
create new place visit widget

### LINKED ISSUE #139: create createPlaceVisitPage

### PR DESCRIPTION
**Pull request by cli1903**

### CODE REVIEW

File: frontend/ui/lib/widgets/create_trip_widget.dart [hunk 1]
```diff
@@ -111,7 +111,7 @@ class _CreateTripWidgetState extends State<CreateTripWidget> {
           child: IconButton(
             onPressed: _onPressed,
             icon: Icon(Icons.send),
-            tooltip: "Click this to submit your trip.",
+            tooltip: "Click this to submit your place to visit.",
```
Comment (anfee1) [inline frontend/ui/lib/widgets/create_trip_widget.dart]: You want this and the create place one switched.

File: frontend/ui/lib/widgets/create_place_visit_widget.dart [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+            ),
+          ),
+        ]),
+        _imageWidget,
+        Padding(
+          padding: const EdgeInsets.all(25.0),
+          child: IconButton(
+            onPressed: _onPressed,
+            icon: Icon(Icons.send),
+            tooltip: "Click this to submit your trip.",
```
Comment (anfee1) [inline frontend/ui/lib/widgets/create_place_visit_widget.dart]: This is what I'm talking about.

File: frontend/ui/lib/pages/trip_view_page.dart [hunk 3]
```diff
@@ -15,12 +16,10 @@ class TripViewPage extends StatelessWidget {
...
       body: Center(
           child: TripViewWidgetFromService(this.tripService, this.tripId)),
-      floatingActionButton: FloatingActionButton(
-        onPressed: () {
-          // TODO: Add place add method to this button
-        },
-        child: Icon(Icons.add),
-        tooltip: "Click to add a place to your trip.",
+      floatingActionButton: FloatingRoutingButton(
+        "/trips/$tripId/placeVisits/new",
```
Comment (anfee1) [inline frontend/ui/lib/pages/trip_view_page.dart]: should maybe add this as a router route

**Review by anfee1 — DISMISSED** [review]
LGTM. tooltip nit should be fixed though.

**Review by KoopaKing — DISMISSED** [review]

**Review by anfee1 — DISMISSED** [review]

**Review by chuckluv — DISMISSED** [review]

**Review by KoopaKing — APPROVED** [review]

**Review by chuckluv — APPROVED** [review]

**Pull request merged**